### PR TITLE
cleanup: fix spelling mistakes in CHANGELOG-1.20.md

### PR DESCRIPTION
**Pull request by wangyx1992**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
This pr fixes spelling mistakes in CHANGELOG-1.20.md.

#### Which issue(s) this PR fixes:
Fixes #

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

### CODE REVIEW

**Review by soniasingla — APPROVED** [review]
/lgtm
/approve

**Comment by wilsonehusin** [comment]
hey @wangyx1992, thanks for submitting this PR. in general, this falls under [trivial edits] which are not encouraged to be merged. I would have an easier time justifying the approval if there are more fixes provided in this PR. would you like to give the 1.20 changelog another check on other spelling issues?

**Comment by wangyx1992** [comment]
> hey @wangyx1992, thanks for submitting this PR. in general, this falls under [trivial edits] which are not encouraged to be merged. I would have an easier time justifying the approval if there are more fixes provided in this PR. would you like to give the 1.20 changelog another check on other spelling issues?

Hi, @wilsonehusin, I have checked the 1.20 changelog and modify found spelling issuses already, waiting for your review with pleasure.

File: CHANGELOG/CHANGELOG-1.20.md [hunk 1]
```diff
@@ -426,7 +426,7 @@ This vulnerability was reported by Rogerio Bastos & Ari Lima from RedHat
   - Add support for IONOS
   
   ### Kubemark
-  - Skip non-kubemark nodes while computing node infos for node groups.
+  - Skip non-kubemark nodes while computing node info for node groups.
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:429]: this should be expanded, i.e.

```suggestion
  - Skip non-kubemark nodes while computing node information for node groups.
```

File: CHANGELOG/CHANGELOG-1.20.md [hunk 2]
```diff
@@ -1137,12 +1137,12 @@ Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provi
...
 - This PR will introduce a feature gate CSIServiceAccountToken with two additional fields in `CSIDriverSpec`. ([#93130](https://github.com/kubernetes/kubernetes/pull/93130), [@zshihang](https://github.com/zshihang)) [SIG API Machinery, Apps, Auth, CLI, Network, Node, Storage and Testing]
-- Users can try the cronjob controller v2 using the feature gate. This will be the default controller in future releases. ([#93370](https://github.com/kubernetes/kubernetes/pull/93370), [@alaypatel07](https://github.com/alaypatel07)) [SIG API Machinery, Apps, Auth and Testing]
+- Users can try the cronJob controller v2 using the feature gate. This will be the default controller in future releases. ([#93370](https://github.com/kubernetes/kubernetes/pull/93370), [@alaypatel07](https://github.com/alaypatel07)) [SIG API Machinery, Apps, Auth and Testing]
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1145]: hmm I think this should be `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md [hunk 3]
```diff
@@ -1295,7 +1295,7 @@ Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provi
   When we switched to go-runner for building the apiserver,
   controller-manager, and scheduler server components, we no longer
   reference the individual architectures in the image names, specifically
-  in the 'FROM' directive of the server image Dockerfiles.
+  in the 'FROM' directive of the server image Dockerfile.
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1298]: I think `Dockerfiles` is right here, plural seem intentional

File: CHANGELOG/CHANGELOG-1.20.md [hunk 4]
```diff
@@ -1439,7 +1439,7 @@ Currently, cadvisor_stats_provider provides AcceleratorStats but cri_stats_provi
 
 ### Other (Cleanup or Flake)
 
-- Handle slow cronjob lister in cronjob controller v2 and improve memory footprint. ([#96443](https://github.com/kubernetes/kubernetes/pull/96443), [@alaypatel07](https://github.com/alaypatel07)) [SIG Apps]
+- Handle slow cronJob lister in cronJob controller v2 and improve memory footprint. ([#96443](https://github.com/kubernetes/kubernetes/pull/96443), [@alaypatel07](https://github.com/alaypatel07)) [SIG Apps]
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1442]: same as previous, `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md [hunk 5]
```diff
@@ -1810,7 +1810,7 @@ filename | sha512 hash
   "action required"
   The feature gate is disabled by default. The user has to enable it for the API Server. ([#94028](https://github.com/kubernetes/kubernetes/pull/94028), [@janosi](https://github.com/janosi)) [SIG API Machinery and Apps]
 - This PR will introduce a feature gate CSIServiceAccountToken with two additional fields in `CSIDriverSpec`. ([#93130](https://github.com/kubernetes/kubernetes/pull/93130), [@zshihang](https://github.com/zshihang)) [SIG API Machinery, Apps, Auth, CLI, Network, Node, Storage and Testing]
-- Users can try the cronjob controller v2 using the feature gate. This will be the default controller in future releases. ([#93370](https://github.com/kubernetes/kubernetes/pull/93370), [@alaypatel07](https://github.com/alaypatel07)) [SIG API Machinery, Apps, Auth and Testing]
+- Users can try the cronJob controller v2 using the feature gate. This will be the default controller in future releases. ([#93370](https://github.com/kubernetes/kubernetes/pull/93370), [@alaypatel07](https://github.com/alaypatel07)) [SIG API Machinery, Apps, Auth and Testing]
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1813]: same as above, i.e. `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md [hunk 6]
```diff
@@ -1894,7 +1894,7 @@ filename | sha512 hash
 
 ### Other (Cleanup or Flake)
 
-- Handle slow cronjob lister in cronjob controller v2 and improve memory footprint. ([#96443](https://github.com/kubernetes/kubernetes/pull/96443), [@alaypatel07](https://github.com/alaypatel07)) [SIG Apps]
+- Handle slow cronJob lister in cronJob controller v2 and improve memory footprint. ([#96443](https://github.com/kubernetes/kubernetes/pull/96443), [@alaypatel07](https://github.com/alaypatel07)) [SIG Apps]
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:1897]: same as above, i.e. `CronJob`

File: CHANGELOG/CHANGELOG-1.20.md [hunk 7]
```diff
@@ -2665,7 +2665,7 @@ filename | sha512 hash
   When we switched to go-runner for building the apiserver,
   controller-manager, and scheduler server components, we no longer
   reference the individual architectures in the image names, specifically
-  in the 'FROM' directive of the server image Dockerfiles.
+  in the 'FROM' directive of the server image Dockerfile.
```
Comment (wilsonehusin) [inline CHANGELOG/CHANGELOG-1.20.md:2668]: same as above, i.e. `Dockerfiles`

**Review by wilsonehusin — CHANGES_REQUESTED** [review]
thanks for taking the time and reviewing the whole document! small changes and then we should be good to go 😄

File: CHANGELOG/CHANGELOG-1.20.md (hunk 3 above)
Comment (wangyx1992) [reply CHANGELOG/CHANGELOG-1.20.md]: Yeah, corresponding to the before `image names`, we should `Dockerfiles` use here.

File: CHANGELOG/CHANGELOG-1.20.md (hunk 7 above)
Comment (wangyx1992) [reply CHANGELOG/CHANGELOG-1.20.md]: Hi, @wilsonehusin, all abolve have been updated already, looking for your review with pleasure.

**Review by wilsonehusin — APPROVED** [review]
awesome, thanks!

/approve

**Pull request merged**